docs/api-testing/import-postman-collection.md

```diff
@@ -123,8 +123,7 @@ Your environmental variables will now be available in the __Variables__ section
 
 Your environmental variables will now available across all your Projects.
 
-## Importing a HAR File from an RDC Job
-<p><span className="sauceDBlue">RDC Only</span></p>
+## Importing a HAR File (RDC and VDC)
```

Comment:
@lauramthomas does it worth to mention what RDC and VDC mean? it seems not all people knows the meaning